TaskTwo/Calculator/Calculator.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+namespace Calculator
+{
+    class Calculator
+    {
+        public bool EnterOperand(out double operand)
+        {
+            Console.WriteLine("Enter number");
+            return double.TryParse(Console.ReadLine(), out operand);
+        }
+
+        public bool Calculate(double firstOperand, double secondOperand, char operationSymbol, ref double result)
+        {
+            bool successFlag = true; 
+            switch (operationSymbol)
+            {
+                case '+':
+                    result = firstOperand + secondOperand;
+                    break;
+                case '-':
+                    result = firstOperand - secondOperand;
+                    break;
+                case '*':
+                    result = firstOperand * secondOperand;
+                    break;
+                case '^':
+                    result = Math.Pow(firstOperand,secondOperand);
+                    break;
+                case '/':
+                    if (secondOperand != 0)
+                    {
+                        result = firstOperand / secondOperand;
+                    }
+                    else
+                    {
+                        successFlag = false;
+                        Console.WriteLine("\nDevision by zero is not allowed");
+                    }
+                    break;
+                default:
+                    successFlag = false;
+                    Console.WriteLine($"\nOperation {operationSymbol} is not supported");
+                    break;
+            }
+            return successFlag;
+        }
+
+        static void Main(string[] args)
+        {
+            Calculator calculator = new Calculator();
+            double firstOperand, secondOperand;
+            if (calculator.EnterOperand(out firstOperand))
```

Comment:
Not needed nested ifs - but the task is completed - we could discuss it after the merge